react/components/EditorContainer/Sidebar/ComponentList/utils.ts

```diff
@@ -1,10 +1,12 @@
 import { SidebarComponent } from '../typings'
 
+import { partition } from 'ramda'
```

Comment:
⛏ This should be the first import, according to our `import` order convention.
```js
import { partition } from 'ramda'

import { SidebarComponent } from '../typings'

import { NormalizedComponent } from './typings'
```

Reply:
I did what you suggest